Snippets/find.sqlpromptsnippet

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-16"?>
```

Comment:
I think this needs to be utf-8. utf-16 doesn't work on my SQL Prompt version (6.6.0.158, which I think is pretty up to date)